region/maridia/outer/Mama Turtle Room.json

```diff
@@ -955,44 +1020,88 @@
       "requires": [
         "canPreciseGrapple",
         "SpaceJump"
+      ]
+    },
+    {
+      "link": [6, 2],
+      "name": "Grapple, Mama Turtle is Sleeping",
+      "requires": [
+        "canUseGrapple",
+        {"obstaclesNotCleared": ["A"]}
       ],
-      "devNote": "Need to consider other transit options: springball, HBJ."
+      "clearsObstacles": ["B"],
+      "note": "Grappling the block does not require canPreciseGrapple if Mama Turtle is asleep, as there is no risk of death on a miss."
```

Comment:
- If you use a in-room strat to reach 6 and can redo that, this is fine.
- If you cross room shinespark, you can usually go right to 2 instead on a retry.

Reply:
It looks like the requirements to spark to 2 vs 6 are the same, except 2 requires an extra ~17 energy. Either can be done tankless if you enter the room.

What would you recommend on this one? I could remove the strat (allowing only the precise grapple variant), add more requirements to it (like gravity+walljump), or leave it as is.
Note that a similar problem could occur if we added a spark to 5 to save energy (which would also require something else, like wall jump, hijump, spring, or space)

Reply:
I think you can leave it as it is